nuget/Microsoft.Windows.CppWinRT.props

```diff
@@ -13,7 +13,6 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </PropertyGroup>
 
     <PropertyGroup>
-        <PreferredToolArchitecture>x64</PreferredToolArchitecture>
```

Comment:
Should this be scoped to the VS2022 build tools (so that VS2019 still gets it set)?

Reply:
Or the opposite; only set it for VS2019 and then VS2022+ do not set it because VS does the right thing already.

Reply:
It's actually a bug to set it in VS2019 when building for ARM64 (reported elsewhere).  And since it's just a build optimization, I think better to remove it altogether, than to effectively copy the intelligence in the VS 2022 Microsoft.Cpp.ToolsetLocation.props.